src/syncer/mock.go

```diff
@@ -20,8 +21,20 @@ func mockSyncer(log *xlog.Log, n int) ([]*Syncer, func()) {
 	var peers []string
 	var httpServers []*http.Server
 	var syncers []*Syncer
+
+	getCurrDir := func() string {
+		dir, err := filepath.Abs(filepath.Dir(os.Args[0]))
+		if err != nil {
+			log.Panicf("mock.getCurrentDirectory err :%+v", err)
```

Comment:
The log style in Radon is module[dot]behaive-with-low-case[dot]err, here shoule be:
mock.getcurrent.directory.err: